website/docs/r/solution_add_on_instance.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_solution_add_on_instance"
+sidebar_current: "docs-vcd-resource-solution-add-on-instance"
+description: |-
+  Provides a resource to manage Solution Add-On Instances in Cloud Director. A Solution Add-On Instance
+  is created from an existing Solution Add-On by supplying configuration values of that particular instance.
+---
+
+# vcd\_solution\_add\_on\_instance
+
+Supported in provider *v3.13+* and VCD 10.4.1+.
+
+Provides a resource to manage Solution Add-Ons Instances in Cloud Director. 
+
+~> Only `System Administrator` can create this resource.
+
+## Example Usage (Uploading an image and create a Solution Add-On entry)
+
+```hcl
+resource "vcd_solution_add_on_instance" "dse14" {
+  add_on_id                     = vcd_solution_add_on.dse14.id
```

Comment:
We need to standartize on `add_on` vs `addon`. I see `addon` already slipped in here too, but maybe it's not too late to change?

<img width="811" alt="Screenshot 2024-06-25 at 15 03 23" src="https://github.com/vmware/terraform-provider-vcd/assets/11330689/84e6518f-85cc-4bd9-b806-83a1b671d8e4">

Reply:
I am for `add_on` because of 
<img width="246" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/15804230/f8987e2a-3043-40e1-b68f-8f7d54dd79d7">

Will attempt to do a search once more

Reply:
Fixed.